.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["react", "prettier"],
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true,
+    "jest": true,
+    "mongo": true,
+    "commonjs": true,
```

Comment:
What do the `commonjs` and `prototypejs` lines do?  It sounds like prototype.js is a JS framework.  Are you using it in this project?